dlp/api/Snippets.Tests/ReidentifyContentUsingFpeTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Xunit;
+
+namespace GoogleCloudSamples
+{
+    public class ReidentifyContentUsingFpeTests : IClassFixture<DlpTestFixture>
+    {
+        private DlpTestFixture _fixture;
+        public ReidentifyContentUsingFpeTests(DlpTestFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public void TestReidentifyWithFpe()
+        {
+            var finding = "731997681";
+            var text = $"My SSN is SSN_TOKEN(9):{finding}";
+            var expectedTransformFake = $"My SSN is {finding}";
+            var result = ReidentifyContentUsingFpe.ReidentifyContent(_fixture.ProjectId, text, _fixture.KeyName, _fixture.WrappedKey);
```

Comment:
avoid using "var" when the type is not easy to infer from surrounding context

Reply:
In the other samples (pre-written), only "var" has been used. So, just to be consistent; I have also used "var" at places.

Reply:
Sure